packages/technical-features/react-application/src/feature.ts

```diff
@@ -13,5 +14,5 @@ export const reactApplicationFeature = getFeature({
     });
   },
 
-  dependencies: [applicationFeature],
+  dependencies: [applicationFeature, randomFeature],
```

Comment:
did you try reverting this and moving it to the top level one?

Reply:
src/main and|or src/renderer? 
I did not.  would you prefer we register them in main &| renderer?

Reply:
I think so because it isn't really a dependency of that feature but more of an implementation of that feature.

Reply:
just checked and moved to main && render